app/javascript/Search/__tests__/SearchFormSync.test.jsx

```diff
@@ -31,9 +31,12 @@ describe('<SearchFormSync />', () => {
   });
 
   it('should synchronize search forms', async () => {
-    const { findByLabelText, findAllByLabelText } = render(<SearchFormSync />, {
-      container: document.body,
-    });
+    // This part of the DOM would be rendered in the search results from the server side.
```

Comment:
These are moved before the render, because the component will render into a div in the document.body while still preserving the markup for the Portal. See https://github.com/testing-library/preact-testing-library/blob/master/src/pure.js#L41

This is the same reason in the other test why it's set before the render.